tests/testthat/test-renderer-geom_raster.R

```diff
@@ -0,0 +1,8 @@
+acontext("geom_raster not defined")
+
+test_that("geom_raster is not defined", {
```

Comment:
please rename this file to test-compiler-geom_raster.R because the renderer is not needed.